recipes-devtools/rust/libstd-rs/rv32.patch

```diff
@@ -0,0 +1,845 @@
+Add RISCV32 support on musl C library
+
+Upstream-Status: Pending
```

Comment:
Do you have a link to the PR?

Reply:
https://github.com/rust-lang/libc/pull/2540

Reply:
Cool! It's probably worth linking it in the commit

Reply:
done